experiments/probabilistic_domain_adaptation/livecell/common.py

```diff
@@ -23,6 +23,7 @@
 from torch_em.util import load_model
 
 CELL_TYPES = ["A172", "BT474", "BV2", "Huh7", "MCF7", "SHSY5Y", "SkBr3", "SKOV3"]
+DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
This is not a good idea to declare globally, and I also don't see that you use it anywhere.